drivers/eaton-ats30-mib.c

```diff
@@ -0,0 +1,370 @@
+/* eaton-ats30-mib.c - subdriver to monitor eaton_ats30 SNMP devices with NUT
+ *
+ *  Copyright (C)
+ *  2011 - 2017	Tomas Halman <TomasHalman@eaton.com>
```

Comment:
Tom probably just generated using my script and replaced the name, not touched the year.
BTW, the copyright is Eaton, and Tom is the author... will fix when merging the PR...